packages/core/src/anchor/memory/in-memory-anchor-service.ts

```diff
@@ -114,7 +115,7 @@ class InMemoryAnchorService implements AnchorService {
           }
           result[candidate.key].push(candidate);
         } catch (e) {
-          console.error(e.message);
+          this.#logger.err(e.message)
```

Comment:
@stbrody Is it possible to pass full error, with stack trace to the log? I think, a stack trace passes valuable information.